docs/fundamentals/executor/executor-api.md

```diff
@@ -263,12 +263,12 @@ with f:
 
 </details>
 
-## Other usages
+## Running Executor outside the Flow
 
-Beside running Executor inside the Flow, we list two other usages that may help you debug.  
-
-### Use Executor out of Flow
+`Executor` objects can be used directly, just like any regular Python object.
+There are two ways of instantiating an Executor object: From a local Python class, and from the Jina Hub.
 
+````{tab} From local Python
```

Comment:
The tabs don't seem to be rendered correctly: 
![image](https://user-images.githubusercontent.com/44071807/156725845-b271b9d7-1acd-4acc-9953-2488f43d2745.png)
![image](https://user-images.githubusercontent.com/44071807/156725913-699cd30b-bdc2-41e8-8460-51d4301198d1.png)

Also, both tabs are called 'local python'

Reply:
Yes Yes I am fixing these !